### DISCUSSION TITLE
p-calendar set to null when date is less than mindate

### DISCUSSION THREAD

**Discussion by ToniSoriano** [primefaces/community#2249] [PrimeNG]
Hello.

The p-calendar component has behavior that is not always optimal.

By default, if it detects that the date is below or above the mindate or maxdate, it stops displaying the date it has in the ngModel.

For our use case it is a problem since there may be previous dates already persisted.

We would need to be able to display those dates and mark the component as invalid, so that the user can see and rectify the date.

Thank you so much.

[Demo stackblitz](https://stackblitz.com/edit/mxvgew-2wxf1x?file=src%2Fapp%2Fcalendar-format-demo.html)

**Comment by Myrrax** [comment]
Running into the same Issue

**Comment by tleveque23** [comment]
Same problem here. And it is a show stopper for us!
